projects/native-ephemery-client-pair-implementation.md

```diff
@@ -0,0 +1,148 @@
+# Project Proposal: 
+# Native Support For Ephemery Testnet Implementation on Client Pair (Besu & Teku) Clients
+
+## Tagline
+Enabling seamless integration of the Ephemery testnet into Ethereum clients with dynamic genesis support.
+
+## Motivation
+Ephemery testnet offers an automatically resetting environment ideal for short-term testing of applications, validators, and client changes. It addresses problems like state bloat and lack of testnet funds in long-running testnets by resetting to genesis, clearing validators, and refilling faucets. This keeps the network small and easy to bootstrap. 
+
+The primary challenge is managing the dynamic genesis, which resets periodically, unlike other networks with hardcoded genesis. 
+
+Integrating Ephemery into clients like Teku and Besu enhances its usability by allowing configuration via a simple flag. This integration promotes experimentation and validation, strengthening the Ethereum ecosystem. 
+
+The project focuses on client configuration and genesis handling, aiming to streamline testnet setup and operation.
+
+## Project Description
+The proposed solution involves implementing native support for the Ephemery testnet in both Teku (an Ethereum consensus client) and Besu (an Ethereum execution client). This will be achieved by introducing a new --ephemery flag that handles the dynamic genesis requirement. 
+
+**The solution includes:**
+- Developing a mechanism within each client to calculate or download the new genesis periodically.
+- Creating an "Ephemery lib" or submodule within each client to handle genesis generation and reset functionality.
+- Ensuring seamless integration with existing client architecture to support Ephemery's unique requirements.
+
+## Specification
+
+The detailed spec can be be found here [eip6916](https://eips.ethereum.org/EIPS/eip-6916)
+
+**Feasibility Study:**
+
+- Assess current client architectures and identify potential integration points for Ephemery.
+- We conducted detailed technical analyses of clients, especially focusing on the reset function.
+- Compile a feasibility report with findings and recommendations.
+
+**Genesis Functionality:**
+
+- Implement a dynamic genesis handling mechanism within both Teku and Besu clients.
+- Develop a library/submodule in Java to manage genesis calculation or retrieval.
+
+**Client Integration:**
+
+- Modify the initialization process in both clients to recognize the --ephemery flag.
+- Ensure that the clients can fetch or compute the genesis block before starting.
+
+**Documentation and Tutorials:**
+- Create comprehensive documentation and tutorials for developers to understand and utilize the new Ephemery functionality.
+
+**Website Development:**
+- Build an Ephemery website featuring a reset countdown, project overview, and documentation.
```

Comment:
Currently there is landing page and resources repo which holds most of this. Another website might be beneficial if done well but I think it would be too much time doing webdev